### PR TITLE
Limit the FOBS error log size

### PR DESCRIPTION
**Pull request by nvidianz**
Fixes FLARE-1140

### Description

Limit the FOBS error message size so it doesn't dump MBs of binary data in the log.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/apis/utils/fl_context_utils.py [hunk 1]
```diff
@@ -20,17 +20,19 @@
...
     new_fl_ctx = FLContext()
     for k, v in fl_ctx.props.items():
         if k not in NonSerializableKeys.KEYS:
             try:
                 fobs.dumps(v)
                 new_fl_ctx.props[k] = v
             except BaseException as e:
-                logger.warning(generate_log_message(fl_ctx, f"Object is not serializable (discarded): {k} - {v}: {e}"))
+                logger.warning(generate_log_message(fl_ctx, "Object is not serializable (discarded): "
+                                                            f"{k} - {type(v)}: {e}"))
```
Comment (YuanTingHsieh) [inline nvflare/apis/utils/fl_context_utils.py:35]: Should we make this more detailed?

"Key {k} in FLContext of type {type(v)} is not serializable (discarded): {e}"

File: nvflare/fuel/utils/fobs/fobs.py [hunk 2]
```diff
@@ -215,7 +215,8 @@ def serialize(obj: Any, **kwargs) -> bytes:
     try:
         return msgpack.packb(obj, default=_fobs_packer, strict_types=True, **kwargs)
     except ValueError as ex:
-        raise ValueError(f"Object {type(obj)} is not serializable: {ex}: {obj}")
+        content = str(obj)[:128]
+        raise ValueError(f"Object {type(obj)} is not serializable: {ex}: {content}")
```
[the remark this answers is not in the document]
Comment (nvidianz) [reply nvflare/fuel/utils/fobs/fobs.py:219]: Changed.

File: nvflare/apis/utils/fl_context_utils.py (hunk 1 above)
Comment (nvidianz) [reply nvflare/apis/utils/fl_context_utils.py:35]: Changed.

**Review by holgerroth — APPROVED** [review]

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**